src/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -148,17 +193,32 @@ private static bool TryFindHierarchyMount(string subsystem, out string root, out
                                 continue;
                             }
 
-                            if (postSeparatorlineParts[0] != "cgroup" ||
-                                Array.IndexOf(postSeparatorlineParts[2].Split(','), subsystem) < 0)
+                            if (!string.Equals(postSeparatorlineParts[0], "cgroup2", StringComparison.Ordinal) &&
+                                (!string.Equals(postSeparatorlineParts[0], "cgroup", StringComparison.Ordinal) ||
```

Comment:
As above, I'm not clear on why the explicit `Equals` call with `Ordinal` is better than just using `!=`.